preview/use-cases.md

```diff
@@ -0,0 +1,98 @@
+
+The [Arcus secret store](https://security.arcus-azure.net/features/secret-store) is the answer. The secret store will be the single point of truth in interacting with required secrets for your application. Multiple 'secret providers' can be configured that accesses the secrets from external sources like: Azure Key Vault, HashiCorp Vault, Docker secrets, environment variables... and the secret store will provide a single access point for the application to fetch those secrets.
+
+An example of how well the secret store integrates within an application:
+
+```csharp
+using Microsoft.Extensions.Hosting;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureAppConfiguration((context, config) => 
+            {
+                config.AddJsonFile("appsettings.json")
+                      .AddJsonFile("appsettings.Development.json");
+            })
+            .ConfigureSecretStore((context, config, builder) =>
+            {
+#if DEBUG
+                builder.AddConfiguration(config);
+#endif
+                var keyVaultName = config["KeyVault_Name"];
+                builder.AddEnvironmentVariables()
+                       .AddAzureKeyVaultWithManagedServiceIdentity($"https://{keyVaultName}.vault.azure.net");
+            });
+    }
+}
+```
+
+For more information, see [our documentation site](https://security.arcus-azure.net/features/secret-store).
+
+Blogs:
+* [Introducing Arcus secret store - making secrets a first-class citizen](https://www.codit.eu/blog/introducing-secret-store-net-core/)
+* [Advanced Arcus secret store features](https://www.codit.eu/blog/secret-store-arcus-security-v1-4/)
+
+## Simplify telemetry tracking with contextual information
+Tracking telemetry with useful information is not an easy task. Giving each telemetry instance enough contextual information, correlated with other telemetry... There are many platforms and logging systems that are doing a part of the work but not one system that combines them all.
+
+[Arcus Observability](https://observability.arcus-azure.net/) is the answer. The library provides a custom [Azure Application Insights Serilog sink](https://observability.arcus-azure.net/Features/sinks/azure-application-insights) that combines the easy application logging from Serilog with the highly expanded contextual information of Azure Application Insights. The library extends Serilog in such a way that all Azure Application Insights features can be logged with the well-known `ILogger` abstraction.
+
+```csharp
+ILogger logger = ...
+
+// Contextual information
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Customer", "Contoso"},
```

Comment:
Maybe include `{ "OrderId", "12345" }` just to remind people that this can be used to track business information

Reply:
Yeah, let's do that! 👍

Reply:
https://github.com/arcus-azure/arcus/pull/204/commits/bd02e8e7be66c7f5592cb406c42573cb40015677